templates/backend/contribution.ts

```diff
@@ -0,0 +1,31 @@
+import { Command, CommandContribution, CommandRegistry} from '@theia/core/lib/common';
+import { inject, injectable } from 'inversify';
+import { HelloBackendWithClientService, HelloBackendService } from '../common/protocol';
+
+const SayHelloViaBackendCommandWithCallBack: Command = {
+    id: 'sayHelloOnBackendWithCallBack.command',
+    label: 'Say hello on the backend with a callback to the client',
+};
+
+const SayHelloViaBackendCommand: Command = {
+    id: 'sayHelloOnBackend.command',
+    label: 'Say hello on the backend',
+};
+
+@injectable()
+export class <%= params.extensionPrefix %>CommandContribution implements CommandContribution {
+
+    constructor(
+        @inject(HelloBackendWithClientService) private readonly helloBackendWithClientService: HelloBackendWithClientService,
+        @inject(HelloBackendService) private readonly helloBackendService: HelloBackendService,
```

Comment:
The injections should be outside the constructor, they align with our coding guidelines:
- https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#property-injection

Reply:
Not consistent with other examples, too, suggest to fix it for all (I can do that):
https://github.com/theia-ide/generator-theia-extension/issues/66